### ISSUE TITLE
Can't support typedoc version 0.21.X

### ISSUE THREAD

**Issue by Kayxue** [felipecrs/typedoc-plugin-rename-defaults#2]
Console output these when I installing this package
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: ***@1.0.0
npm ERR! Found: typedoc@0.21.5
npm ERR! node_modules/typedoc
npm ERR!   dev typedoc@"^0.21.5" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer typedoc@"0.20.x" from typedoc-plugin-rename-defaults@0.1.0
npm ERR! node_modules/typedoc-plugin-rename-defaults
npm ERR!   dev typedoc-plugin-rename-defaults@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\***\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\***\AppData\Local\npm-cache\_logs\2021-08-07T10_27_27_319Z-debug.log
```

**Comment by brandonmpetty** [comment]
Same issue.  I can't believe this plugin is not mainlined into Typedoc.  At least as a param setting.
Most of my classes and interfaces are "default"... and no one wants to see "default" everywhere.

Will this plugin need to be updated with every release of Typedoc?

**Comment by felipecrs** [comment]
I'm gonna take a look at this. As TypeDoc is still in 0.x, every minor update may contain breaking changes, that's why the version of TypeDoc accepted as peer dependency in the package.json needs to be updated manually after every new release of TypeDoc. Once the API is finalized (1.0), most likely it will work until TypeDoc reaches 2.0.


Btw, PRs are welcome.

**Comment by felipecrs** [comment]
Fixed with 8473e61d312138837169dacb2adf39530ac1f353, released as [v0.2.0](https://www.npmjs.com/package/typedoc-plugin-rename-defaults/v/0.2.0).
